tests/datavolume_test.go

```diff
@@ -1013,7 +1013,7 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 				runningCondition: &cdiv1.DataVolumeCondition{
 					Type:    cdiv1.DataVolumeRunning,
 					Status:  v1.ConditionFalse,
-					Message: "Import Complete; VDDK: {\"Version\":\"1.2.3\",\"Host\":\"esx.test\"}",
```

Comment:
Probably missing something trivial, why are we dropping this from the expected output, shouldn't the version and host get propagated correctly?

Reply:
After importing the importer Pod writes the extended termination message to its termination log. Once the import controller sees it and was sucessfully able to parse the message, the value written to the message annotation will be just `Import Complete` instead of the full serialized JSON string. The import controller will still set the appropriate annotations for VDDK version and host.

Reply:
I see. @mrnold hey! do you happen to know if anyone was treating this message as an API on the CDI consuming side?